src/cares_wrap.cc

```diff
@@ -912,14 +883,12 @@ void AfterGetAddrInfo(uv_getaddrinfo_t* req, int status, struct addrinfo* res) {
   HandleScope handle_scope(env->isolate());
   Context::Scope context_scope(env->context());
 
-  Local<Value> argv[] = {
-    Integer::New(env->isolate(), status),
-    Null(env->isolate())
-  };
+  Local<Value> argv[] = {Integer::New(env->isolate(), status),
```

Comment:
I think the old style is probably more readable by many of the contributors here b/c most (including myself) have more JS experience than C++, and the previous style more closely matches how it's usually done in JS.

Not arguing for or against. Just stating an observation.